populate.py

```diff
@@ -28,21 +28,17 @@
 
 def setup_config(network='docker'):
     print('Init config for owkin and chunantes')
+    USER, PASSWORD = ('substra', 'p@$swr0d44')
     if network == 'docker':
-        client.add_profile('owkin', 'http://owkin.substra-backend:8000', '0.0',
-                           user=USER, password=PASSWORD)
-        client.add_profile('chunantes', 'http://chunantes.substra-backend:8001', '0.0',
-                           user=USER, password=PASSWORD)
-        client.add_profile('clb', 'http://clb.substra-backend:8002', '0.0',
-                           user=USER, password=PASSWORD)
+        # get first available user
+        client.add_profile('owkin', USER, PASSWORD, 'http://substra-backend.owkin.xyz:8000', '0.0')
+        client.add_profile('chunantes', USER, PASSWORD, 'http://substra-backend.chunantes.xyz:8001', '0.0')
+        client.add_profile('clb', USER, PASSWORD, 'http://substra-backend.clb.xyz:8002', '0.0')
     if network == 'skaffold':
         # the usernames and passwords are defined in the skaffold.yaml file
-        client.add_profile('owkin', 'http://substra-backend.node-1', '0.0',
-                           user='node-1', password='node-1pwd')
-        client.add_profile('chunantes', 'http://substra-backend.node-2', '0.0',
-                           user='node-2', password='node-2pwd')
-        client.add_profile('clb', 'http://substra-backend.node-3', '0.0',
-                           user='node-3', password='node-3pwd')
+        client.add_profile('owkin', USER, PASSWORD, 'http://substra-backend.node-1.com', '0.0')
+        client.add_profile('chunantes', USER, PASSWORD, 'http://substra-backend.node-2.com', '0.0')
+        client.add_profile('clb', USER, PASSWORD, 'http://substra-backend.node-3.com', '0.0')
```

Comment:
The skaffold conf was using separate user/pwd for each node, this uses the same for all nodes (similar to the docker compose profile). This could lead to issues being not detected because the same credentials could be used to the same nodes. 

I think we should keep things as they were (different user/pwd for skaffold and the just one for the docker-compose setup) and then in a separate PR decide on how to harmonize both (I'd be for different user/pwd in all environments).

Reply:
Yes I will try to modify that.
It was easier to test right now with the same user, but definitively we should add different users.
It will require extra new code though with the job `add_user` and the `add_user` management command used in the start.py.

Reply:
👍

Reply:
updated, it works great